src/official.md

```diff
@@ -22,6 +22,7 @@ Advanced:
 
 Other:
 * [Asynchronous Programming in Rust](https://rust-lang.github.io/async-book/)
+* [The Chalk Book](https://rust-lang.github.io/chalk/book/) - Rust's new trait system implementation
```

Comment:
I would normally use the title "What is Chalk?" as the link, but "The Chalk Book" is so much more sensible.